lib/stella/helpers.rb

```diff
@@ -0,0 +1,103 @@
+require 'typhoeus/adapters/faraday'
+
+module Stella
+  module Helpers
+    extend ActiveSupport::Concern
+
+    @@types = []
+
+    included do
+      if defined? Rails
```

Comment:
I'm going to remove the `env` from the naming scheme altogether and just provide the override hook to client app. I will update the readme with info on that.